tm4c-hal/src/i2c.rs

```diff
@@ -60,26 +62,45 @@ macro_rules! i2c_busy_wait {
             .write(|w| unsafe { w.cntl().bits((1_000 >> 4) as u8) });
 
         let mcs = loop {
-            let mcs = $i2c.mcs.read();
+                let mcs = $i2c.mcs.read();
+
+                if mcs.busy().bit_is_clear() {
+                    break mcs;
+                }
+            };
 
-            if mcs.busy().bit_is_clear() {
-                break mcs;
-            }
-        };
 
         if mcs.clkto().bit_is_set() {
-            Err(Error::Timeout)
-        } else if mcs.busbsy().bit_is_set() {
-            Err(Error::BusBusy)
-        } else if mcs.arblst().bit_is_set() {
-            Err(Error::Arbitration)
-        } else if mcs.datack().bit_is_set() {
-            Err(Error::DataAck)
-        } else if mcs.adrack().bit_is_set() {
-            Err(Error::AdrAck)
-        } else {
-            Ok(())
-        }
+                return Err(Error::Timeout)
```

Comment:
Also extra indentation here. It's annoying rustfmt won't format inside a macro.

Reply:
Sorry, didn't know rustfmt does not indent macro, I was convinced that everything was correct after using it.